temba/channels/views.py

```diff
@@ -1554,12 +1554,16 @@ def form_valid(self, form):
                 raise Exception(_("No org for this user, cannot claim"))
 
             data = form.cleaned_data
+            if not data.get('api_url'):
+                api_url = 'https://go.vumi.org/api/v1/go/http_api_nostream'
+            else:
+                api_url = data['api_url']
```

Comment:
Same default get as above.. (I think, can't remember if Django includes the key if not present as an optional field)